Resources/doc/index.md

```diff
@@ -5,143 +5,14 @@ LdapBundle provides a Ldap authentication system without Apache's `mod_ldap` mod
-          always_use_default_target_path: true
-          default_target_path: /profile
-      logout:     true
-      anonymous:  true
-
-  providers:
-    fr3d_ldapbundle:
-      id: fr3d_ldap.security.user.provider
-
-  encoders:
-      AcmeBundle\Acme\User\LdapUser: plaintext
-```
-
-**Optional you can also enable basic authentication**
-``` yaml
-# app/config/security.yml
-
-security:
-  firewalls:
-    api:
-      pattern:    ^/api
-      fr3d_ldap_httpbasic: ~
-```
-
-### 4. Configure config.yml
-``` yaml
-# app/config/config.yml
-fr3d_ldap:
-    driver:
-        host:                your.host.foo
-#       port:                389    # Optional
-#       username:            foo    # Optional
-#       password:            bar    # Optional
-#       bindRequiresDn:      true   # Optional
-#       baseDn:              ou=users, dc=host, dc=foo   # Optional
-#       accountFilterFormat: (&(uid=%s)) # Optional. sprintf format %s will be the username
-#       optReferrals:        false  # Optional
-#       useSsl:              true   # Enable SSL negotiation. Optional
-#       useStartTls:         true   # Enable TLS negotiation. Optional
-#       accountCanonicalForm: 3 # ACCTNAME_FORM_BACKSLASH this is only needed if your users have to login with something like HOST\User
-#       accountDomainName: HOST
-#       accountDomainNameShort: HOST # if you use the Backslash form set both to Hostname than the Username will be converted to HOST\User
-    user:
-        baseDn: ou=users, dc=host, dc=foo
-        filter: (&(ObjectClass=Person))
-        attributes:          # Specify ldap attributes mapping [ldap attribute, user object method]
-#           - { ldap_attr: uid,  user_method: setUsername } # Default
-#           - { ldap_attr: cn,   user_method: setName }     # Optional
-
-#           - { ldap_attr: ...,  user_method: ... }         # Optional
-#   service:
-#       user_manager: fos_user.user_manager          # Overrides default user manager
-#       ldap_manager: fr3d_ldap.ldap_manager.default # Overrides default ldap manager
-```
-
-**You need to configure the parameters under the fr3d_ldap section.**
-
-### 5. Enable FOSUserBundle as User Provider
-
-In security.yml make a `chain_provider` with `fos_userbundle` before `fr3d_ldapbundle` .
-
-``` yaml
-# app/config/security.yml
-
-security:
-    providers:
-        chain_provider:
-            chain:
-                providers: [fos_userbundle, fr3d_ldapbundle]
-
-        fr3d_ldapbundle:
-            id: fr3d_ldap.security.user.provider
-
-        fos_userbundle:
-            id: fos_user.user_provider.username
-
-```
-
-Make sure to set all needed fields when creating the user otherwise you will get an exception. This can easily be done in the config.yml
-
-``` yaml
-# app/config/config.yml
-fr3d_ldap:
-    ...
-    user:
-      - { ldap_attr: uid,  user_method: setUsername }
-      - { ldap_attr: mail, user_method: setEmail }
-```
+1. [Standalone install](install/standalone.md)
```

Comment:
This shouldn't be an ordered list.

Ordered list implies after complete step 1 continue with step 2.